pkg/csource/csource.go

```diff
@@ -269,6 +270,8 @@ func (ctx *context) emitCallName(w *bytes.Buffer, call prog.ExecCall, native boo
 		fmt.Fprintf(w, "syscall(%v%v", ctx.sysTarget.SyscallPrefix, callName)
 	} else if strings.HasPrefix(callName, "syz_") {
 		fmt.Fprintf(w, "%v(", callName)
+	} else if trampolineName, ok := ctx.sysTarget.SyscallTrampolines[callName]; ok {
```

Comment:
It somehow works without the cast below?
mmap accepts some pointers and we will pass integers... am I missing something?

Reply:
It indeed compiles but with warnings, furthermore, include <sys/mman.h> is missing, compiler complains about undefined function. Worth providing a PR for this ?

Reply:
I think it would be good to change callName in the beginning of the function to the trampoline. Then (1) we can use "syz_" function as trampoline as well, (2) we will get the cast logic for free.

Re <sys/mman.h>. I think the simplest solution for now is to include it in common_linux.h under GOOS_s390x, then it will get into reproducers automatically.